src/modules/quizzes.js

```diff
@@ -40,16 +40,21 @@ function createQuizUrl(quizId, parameters, options, path) {
   }
 
   if (parameters) {
-    const { section, answers, versionId } = parameters;
+    const { section, answers, versionId, quizSessionId, quizVersionId = versionId } = parameters;
```

Comment:
I think it's fine to make this change breaking and just use `quizVersionId` here. We just need to check if `demo` uses `versionId` if not, we can drop `versionId` so we don't have to maintain it

Reply:
👍 I don't believe we send version_id in the quiz so I will get rido f this.